lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
I need something which can return the base and the exponent of the power, which is now done in `ConstantTerms`. This is needed to collect the exponents and getting the value of the base. I'm not quite sure how to do this in another way or where to place the code for that if not in `ConstantTerms`. I rewrote it a little bit and deleted lots of unnecessary code in it. So now it creates a power node with base and exponent if all terms are constant and the operation is multiplication.